samples/bidirectional-chat/README.md

```diff
@@ -0,0 +1,104 @@
+# Azure function bidirectional chatroom sample
+
+This sample demonstrate how to integrate SignalR Service upstream feature with Azure Function. This features SignalR Service sending messages to upstream endpoints in serverless scenario. With the signalr function binding, you can leverge class based model in C# that can give you a consistent experience with non-serverless scanarios.
```

Comment:
`demonstrate` => `demonstrates`

I feel the first line introduces in new terms such as "upstream", which might confuse people from the very beginning. How about having a simple introduction, that it is a complete fully functional sample that can have chatroom running towards Azure Function?